src/pages/HoF.js

```diff
@@ -52,6 +52,11 @@ const HallOfFame = () => {
         name: 'Isaac Zarsky',
         dates: 'February 2018 - May 2020',
         pic: 'https://github.com/zarsky-broad.png?size=200'
+      },
+      {
+        name: 'Adrian Sharma',
+        dates: 'October 2019 - May 2020',
+        pic: 'https://files.slack.com/files-pri/T0CMFS7GX-F014HJFR3TL/adrian.jpeg'
```

Comment:
This also doesn't work for me. I was able to find this: `https://avatars.slack-edge.com/2018-10-29/468158425702_59279567a73f8515a939_192.jpg` by opening slack in Chrome, getting a token from a request in the network inspector, and using this: `https://api.slack.com/methods/users.profile.get/test`.

Reply:
shoot okay I will fix it